docs/user/basics/documents-creator.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Documents Creator
+description: Create user-friendly documents
+lang: en
+---
+
+# Documents Creator
+
+The Documents creator converts your documentation data into user-friendly documents. This allows you to reach a diverse audience - from tech-savvy stakeholders to the less digitally fluent decision-makers.
+
+The documents can serve as a documentation basis for audits, conformity checks and customer reporting. Documents are available in the main navigation bar.
+
+## Rights
+
+You should check the [Rights](../../admin/rights-and-permissions.md) and alter them to fit your needs. It is also possible to set `All available Rights (inlcuding future updates)`.
```

Comment:
We have a variaty of rights know which can be described here:

* Download
* Manage documents
* Manage mailings